nilearn/_utils/ndimage.py

```diff
@@ -38,6 +47,40 @@ def largest_connected_component(volume):
     return labels == label_count.argmax()
 
 
+def largest_connected_component_img(imgs):
+    """ Return the largest connected component of an image or list of images.
+
```

Comment:
Can you add this here ?  ```.. versionadded:: 0.3``` with empty line space before and after.

In Nilearn, you can have a look at this link
https://github.com/nilearn/nilearn/blob/230279e885aae8f9ae28c99c291ed3b0b4e93588/nilearn/connectome/connectivity_matrices.py#L204

Ref for version added and other: http://www.sphinx-doc.org/en/stable/markup/para.html